test/test_basic.py

```diff
@@ -411,17 +418,17 @@ def test_with_local_changes(mc):
     assert any(local_changes.values()) is False
     dbsync_status(mc)
 
+@pytest.mark.parametrize("project_name", ['test_recreated_project_ids', 'Test_Recreated_Project_Ids'])
```

Comment:
do we actually need to parametrize all these test cases? it seems like a lot of extra testing - I would rather just add one extra test case to test schema escaping (just checking that init+push+pull would not break)

Reply:
I removed the parameters and created one simpler test to try different project names.